testing/util.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+///////////////////////////////////////////////////
+// This script is intended to be run as a cron job
+// every n-minutes or whatever the equiv is on windows
+//
+// Author: John A. [euclidjda](https://github.com/euclidjda)
```

Comment:
It was already in the `populate.js` file. I spit some functions to the `util.js`, and therefore I copied the author names along. I didn't want to be the person who removes the names of the original author, but I agree that in an open-source project author names are not appropriate in the source itself.